rtribbia/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
Looks good throughout! I've given you some feedback on the subclassing exercise where you asked for it. Feel free to get in touch if you still have questions about that, or with anything else.